docs/docs/helm.mdx

```diff
@@ -142,7 +142,7 @@ to add it:
 version: 1
 repositories:
   - name: stable
-    url: https://kubernetes-charts.storage.googleapis.com
+    url: https://charts.helm.sh/stable
 + - name: grafana
 +   url: https://https://grafana.github.io/helm-charts
```

Comment:
I got distracted by this diff, decided to fix it in https://github.com/grafana/tanka/pull/426